recipes/nbcommands/meta.yaml

```diff
@@ -0,0 +1,53 @@
+{% set name = "nbcommands" %}
+{% set version = "0.3.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: "https://github.com/vinayak-mehta/nbcommands/archive/v{{ version }}.tar.gz"
+  sha256: 77c6911e7171a2cabb584159e898a7932e90613512b0cfec8744055a4d3b1985
+
+build:
+  noarch: python
+  number: 0
+  entry_points:
+    - nbblack = nbcommands._black:_black
+    - nbtouch = nbcommands._touch:touch
+    - nbgrep = nbcommands._grep:grep
+    - nbhead = nbcommands._head:head
+    - nbtail = nbcommands._tail:tail
+    - nbcat = nbcommands._cat:cat
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
+
+requirements:
+  host:
+    - python
+    - pip
+  run:
+    - python
+    - black
```

Comment:
because it relies on ``black`` I think this project has to be ``Python >= 3.6``

Reply:
I tried doing that earlier with a `skip: True  # [py<36]` but the linter raised an error.